addons/account_debit_note/views/account_debit_note_menus.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+    <record id="debit_notes_customer_action" model="ir.actions.act_window">
```

Comment:
@chklop What do you think about adding a separate menu for the Debit Notes?

Reply:
Is that a separate menu, like Accounting > Customer > Debit Notes ?

Reply:
yes @chklop